tests/brevitas/nn/test_recurrent.py

```diff
@@ -130,6 +136,13 @@ def test_lstm_quant_disabled_fwd(
         # cell states
         assert torch.isclose(out[1][1], ref_out[1][1], atol=ATOL).all()
 
+        # Test that the brevitas model can be saved/loaded without warning
```

Comment:
Could you please move this assert to a new test called test_quantlstm_state_dict where you instantiate a single QuantLSTM with constant inp_size/hidden_size/etc. where you check only this assert?